docs/source/getting-started/installation.rst

```diff
@@ -62,8 +64,11 @@ To get the **AWS access key** required by :code:`aws configure`, please go to th
   $ # Configure your AWS credentials
   $ aws configure
 
+To create a new AWS user with minimal permissions for SkyPilot, see :ref:`AWS User Creation <cloud-permissions-aws>`.
```

Comment:
Can we add something like **Optional** and/or "not needed to get started with SkyPilot" disclaimer?

I am worried that it sounds like required steps.

Perhaps put it below L69.

Same for GCP?